websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,164 @@
+				"[class*=VideoSummary_programLink] picture img"
+			).src;
+		} else presenceData.largeImageKey = Assets.TFX;
+	} else if (path.startsWith("/tmc/direct")) {
+		presenceData.details = document.querySelector(
+			"[class*=VideoSummary_programName]"
+		).textContent;
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Stream;
+		presenceData.smallImageText = live;
+		if (
+			document.querySelector<HTMLImageElement>(
+				"[class*=VideoSummary_programLink] picture img"
+			).src
+		) {
+			presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+				"[class*=VideoSummary_programLink] picture img"
+			).src;
+		} else presenceData.largeImageKey = Assets.TMC;
+	} else if (path.startsWith("/tf1-series-films/direct")) {
+		presenceData.details = document.querySelector(
+			"[class*=VideoSummary_programName]"
+		).textContent;
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Stream;
+		presenceData.smallImageText = live;
+		if (
+			document.querySelector<HTMLImageElement>(
+				"[class*=VideoSummary_programLink] picture img"
+			).src
+		) {
+			presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+				"[class*=VideoSummary_programLink] picture img"
+			).src;
+		} else presenceData.largeImageKey = Assets.TF1Series;
+	} else if (path.startsWith("/lci/direct")) {
+		presenceData.details = "La Chaîne Info";
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Stream;
+		presenceData.smallImageText = live;
+		presenceData.largeImageKey = Assets.LCI;
+	} else if (path.startsWith("/stream")) {
+		presenceData.details = `Stream : ${
+			document.querySelector("[class*=VideoSummary_programName]").textContent
+		}`;
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Stream;
+		presenceData.smallImageText = live;
+	} else if (path.includes("/programmes-tv")) {
+		presenceData.details = "Programmes TV";
+		presenceData.state = `${browse} Liste des programmes`;
+		if (path.split("/")[1] === "programmes-tv")
+			presenceData.largeImageKey = Assets.MyTF1;
+		else presenceData.largeImageKey = path.split("/")[1];
+	} else if (path.startsWith("/mon-compte")) {
+		presenceData.details = viewProfile;
+		presenceData.state = browse;
+	} else if (
+		path.split("/")[3].includes("videos") &&
+		path.split("/")[4].includes("-")
+	) {
+		presenceData.details = document.querySelector(
+			"[class*=VideoSummary_programName]"
+		).textContent;
+		presenceData.state = document.querySelector(
+			"[class*=VideoSummary_title]"
+		).textContent;
+		presenceData.smallImageKey = Assets.Replay;
+		presenceData.smallImageText = watchingVid;
+		presenceData.largeImageKey = path.split("/")[1];
```

Comment:
The `largeImageKey` fields here should use Imgur links/assets rather than Discord asset names.